icu4c/source/common/unicode/localebuilder.h

```diff
@@ -279,6 +279,7 @@ class U_COMMON_API LocaleBuilder : public UObject {
      */
     Locale build(UErrorCode& status);
 
+#ifndef U_HIDE_DRAFT_API
```

Comment:
FYI Not strictly necessary. I understand this helps with drafts from different versions. My tool should find problems when we promote draft 64 but not 65 and forget this here.